system/doc/design_principles/release_handling.md

```diff
@@ -24,7 +24,8 @@ limitations under the License.
 ## Release Handling Principles
 
 An important feature of the Erlang programming language is the ability to change
-module code in runtime, _code replacement_, as described in the Erlang Reference
+module code in runtime, _code replacement_, as described in
```

Comment:
```suggestion
module code at runtime, _code replacement_, as described in
```
(I think)

Reply:
Yes, "at runtime" seems to be the correct phrasing.